.github/workflows/test.yml

```diff
@@ -37,6 +37,8 @@ jobs:
           (apt-get update) && (apt-get upgrade)
           apt-get -y install python3-pip
           pip3 install neclib
+          cd ros2_ws/src/
+          git clone https://github.com/necst-telescope/necst-msgs.git
```

Comment:
This package won't experience `colcon build`, which may cause `ImportError`

Reply:
Ooopps, sry
Nice review!!